lib/statix.ex

```diff
@@ -313,6 +313,14 @@ defmodule Statix do
       def set(key, val, options \\ []) do
         Statix.transmit(current_conn(), :set, key, val, options)
       end
+
+      defoverridable [increment: 1, increment: 2, increment: 3,
```

Comment:
Maybe we should make only the top-arity functions overridable?

Reply:
Agreed!

Reply:
Done! 👍